views/layouts/main.handlebars

```diff
@@ -32,8 +32,8 @@
       <a href="https://www.mozilla.org"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>
       <a href="https://www.mozilla.org/about/legal" data-l10n-id="footerLinkLegal"></a>
       <a href="https://testpilot.firefox.com/about" data-l10n-id="footerLinkAbout"></a>
-      <a href="https://testpilot.firefox.com/privacy" data-l10n-id="footerLinkPrivacy"></a>
-      <a href="https://testpilot.firefox.com/terms" data-l10n-id="footerLinkTerms"></a>
+      <a href="/legal" data-l10n-id="footerLinkPrivacy"></a>
+      <a href="/legal" data-l10n-id="footerLinkTerms"></a>
```

Comment:
Whatever gets it shipped is fine w/ legal.  Already a patch for this, so this works for me. :)